src/helm/benchmark/metrics/ifeval_instructions.py

```diff
@@ -0,0 +1,1500 @@
+# coding=utf-8
+# Copyright 2023 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Library of instructions."""
+import collections
+import json
+import random
+import re
+import string
+from typing import Dict, Optional, Sequence, Union
+
+from absl import logging
+import langdetect
+
+import src.helm.benchmark.metrics.ifeval_instructions_util as instructions_util
+
+from src.helm.benchmark.metrics.ifeval_instructions_util import LANGUAGE_CODES
```

Comment:
Delete `src.` from these imports.

Likewise for the other files.

Reply:
Addressed in the latest change.